### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by VasilMitovClouway**
Factory task ready for review.

### CODE REVIEW

File: src/com/clouway/codecfactory/AbstractFactory.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.clouway.codecfactory;
+
+/**
+ * @author Vasil Mitov <v.mitov.clouway@gmail.com>
+ */
+public abstract class AbstractFactory {
```
Comment (freekman) [inline src/com/clouway/codecfactory/AbstractFactory.java]: tell me what is the difference between Factory and AbstractFactory

Comment (VasilMitovClouway) [inline src/com/clouway/codecfactory/AbstractFactory.java]: With the Factory we create concrete implementations (Audi Ford BMW) of an interface (Car). With AbstractFactory we create implementations of different Factorys that in terms know how to create different implementations of that interface.

Comment (VasilMitovClouway) [inline src/com/clouway/codecfactory/AbstractFactory.java]: I wasnt sure should i make AbstractFactory make XMLFactory and JSONFactory or the thing that i pushed in the PR.

**Comment by freekman** [comment]
LGTM

**Pull request merged**